src/main/scala/firrtl/Namespace.scala

```diff
@@ -21,23 +22,23 @@ class Namespace private {
 
   def contains(value: String): Boolean = namespace.contains(value)
 
-  private def newNameIndex(value: String, idx: Int): (String, Int) = {
-    var n = idx
-    var str = value
-    while (!tryName(str)) {
-      str = s"${value}_$n"
-      n += 1
+  def newName(value: String): String = {
+    // First try, just check
+    if (tryName(value)) value
+    else {
+      var idx = indices.getOrElse(value, 0)
+      var str = value
+      do {
+        str = s"${value}_$idx"
+        idx += 1
+      }
+      while (!(tryName(str)))
```

Comment:
Is this the right style? I always put these on the same line as closing brackets

Reply:
Yeah I usually put it on the same line as well, I just goofed